tapdb/universe_perf_test.go

```diff
@@ -180,19 +181,20 @@ func formatSize(bytes int64) string {
 
 // TestUniverseIndexPerformance tests query performance with indices.
 func TestUniverseIndexPerformance(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping index performance test in short mode")
-	}
+    if testing.Short() {
+        t.Skip("skipping index performance test in short mode")
+    }
 
-	t.Parallel()
+    t.Parallel()
 
-	const (
-		numAssets         = 50
-		numLeavesPerTree  = 10
-		numEventsPerAsset = 25
-		numQueries        = 50
-		batchSize         = 5
-	)
+    // Check for universe-scale test flag
+    if flag.Lookup("test.universe-scale") != nil {
```

Comment:
Nice. I think we should rename the flag though, since anything `test.*` is kind of reserved for Golang internal testing harness. So can perhaps just be `long-tests` and we can re-use that flag for other places too?